core/Command/Maintenance/RepairShareOwnership.php

```diff
@@ -77,7 +77,7 @@ protected function execute(InputInterface $input, OutputInterface $output): int
 			$output->writeln("Found " . count($shares) . " shares with invalid share owner");
 			foreach ($shares as $share) {
 				/** @var array{shareId: int, fileTarget: string, initiator: string, receiver: string, owner: string, mountOwner: string} $share */
-				$output->writeln(" - share ${share['shareId']} from \"${share['initiator']}\" to \"${share['receiver']}\" at \"${share['fileTarget']}\", owned by \"${share['owner']}\", that should be owned by \"${share['mountOwner']}\"");
+				$output->writeln(" - share {$share['shareId']} from \"{$share['initiator']}\" to \"{$share['receiver']}\" at \"${share['fileTarget']}\", owned by \"{$share['owner']}\", that should be owned by \"{$share['mountOwner']}\"");
```

Comment:
```suggestion
				$output->writeln(" - share {$share['shareId']} from \"{$share['initiator']}\" to \"{$share['receiver']}\" at \"{$share['fileTarget']}\", owned by \"{$share['owner']}\", that should be owned by \"{$share['mountOwner']}\"");
```
Was that a test to see how closely I look at what I review :stuck_out_tongue: